dream.py

```diff
@@ -130,7 +194,7 @@ def _make_cylinder_coords(self, n_pixel, n_rows=1, height=0.1, radius=4,
 
         return pixel_coords
 
-    def generate_data(self, n_events):
+    def generate_data_test(self, n_events):
```

Comment:
Is this function still useful? It was meant to demo the API for adding sparse data, but can probably be removed now that you have a proper generator?

Reply:
It can be removed, will update the branch.